.github/workflows/anonymization-backend-release.yml

```diff
@@ -0,0 +1,125 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      title:
+        description: 'Title for the release'
+        required: true
+        type: string
+
+      description:
+        description: 'Human-readable description of the release'
+        required: true
+        type: string
+
+      ref:
+        description: 'Commit to base the release on (defaults to `HEAD`)'
+        required: false
+        type: string
+
+      tag:
+        description: 'Tag for the release (defaults to `anonymization-backend-<COMMIT>`)'
+        required: false
+        type: string
+
+permissions:
+  contents: write
+
+env:
+  NAME: anonymization-backend
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  ok:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Bazel environment
+        uses: bazel-contrib/setup-bazel@0.9.1
+
+      - name: build
+        run: |
+          TARGET='//rs/boundary_node/anonymization/backend:anonymization_backend'
+          bazel build --config=ci ${TARGET}
+
+          OUTPUT='bazel-bin/rs/boundary_node/anonymization/backend/anonymization_backend.wasm.gz'
+          mv ${OUTPUT} anonymization_backend.wasm.gz
+
+          ARTIFACTS=(
+            anonymization_backend.wasm.gz
+          )
+
+          echo "ARTIFACTS=${ARTIFACTS[@]}" >> "${GITHUB_ENV}"
+
+      - name: checksums
+        run: |
+          CHECKSUMS=$(mktemp)
+
+          for ARTIFACT in ${ARTIFACTS[@]}; do
+              shasum -a256 ${ARTIFACT} >> ${CHECKSUMS}
+          done
+
+          echo "CHECKSUMS=${CHECKSUMS}" >> "${GITHUB_ENV}"
+
+      - name: tag
+        run: |
+          REF='${{ inputs.ref }}'
```

Comment:
A cleaner way: 
```
REF=${{ inputs.ref || github.ref }}
```

Reply:
@cgundy @basvandijk

I'm not sure I fully understand, is this what you mean?

```
jobs:
  job:
    - uses: actions/checkout@v4
       with:
         ref: ${{ inputs.ref | github.sha }}
    
    - name: tag
        run: |
          RELEASE_TAG='${{ inputs.tag }}'
          if [[ -z "${RELEASE_TAG}" ]]; then
            COMMIT_SHORT=$(git rev-parse --short HEAD)
            RELEASE_TAG="${{ env.NAME }}-${COMMIT_SHORT}"
          fi
```

I have to say that personally, this spreads this logic a bit over the file instead of concentrating it in that one place, so I think I'd prefer the current way - but let me know if I'm just missing something...

Reply:
First, the checkout logic needs to happen before the build, otherwise it will not run on the ref you want it to run on. And you can keep
```
    - uses: actions/checkout@v4
       with:
         ref: ${{ inputs.ref }}
```
If inputs is empty it will automatically default to the commit the workflow was triggered on, which on a manual trigger case will always be the head of master.

Reply:
Ok, based on our offline discussion I ended up removing the ability to specify a commit.